@AH-64D Apache Longbow/Addons/fza_ah64_controls/config/CfgAmmo.hpp

```diff
@@ -420,6 +458,33 @@ class CfgAmmo
 
 		//AI usage
 		cost 						= 500;
+        class ace_missileguidance {
+            enabled 							= 0;
+
+            minDeflection 						= 0.0005;      // Minium flap deflection for guidance
```

Comment:
This is a lot of modifications and redundancy in CfgAmmo. You should be albe to use [Class Inheritance](https://community.bistudio.com/wiki/Class_Inheritance) to reduce it